models/members.js

```diff
@@ -5,21 +5,24 @@
 
 const firestore = require("../utils/firestore");
 const userModel = firestore.collection("users");
-
+const { ROLES } = require("../constants/roles");
 /**
  * Fetches the data about our users
  * @return {Promise<userModel|Array>}
  */
 
-const fetchUsers = async () => {
+const fetchUsers = async (queryParams = {}) => {
   try {
     const snapshot = await userModel.get();
```

Comment:
This doesn't do what we discussed, about fetching only the users who have `archived != true`

Reply:
Created a new issue and PR for this : https://github.com/Real-Dev-Squad/website-backend/pull/656